packages/DataTable/src/hooks/useSticky.ts

```diff
@@ -0,0 +1,44 @@
+import { Cell, ColumnInstance, Hooks, TableCellProps, TableCommonProps, TableHeaderProps } from "react-table";
+
+const stickyCellStyles = {
+  backgroundColor: "inherit",
+  position: "sticky",
+  zIndex: 1,
+};
+
+interface Column extends ColumnInstance {
+  isSticky?: boolean;
+}
+
+function isColumnSticky(column: Column): boolean {
+  if (column.isSticky === true) {
```

Comment:
wouldn't `if (column.isSticky) {` just be enough? 
or at least `if (Boolean(column.isSticky)) {`

Reply:
also `return true` would be easier to read than `return column.isSticky;` IMO